lib/ansible/galaxy/collection.py

```diff
@@ -1070,46 +1167,128 @@ def _build_dependency_map(collections, existing_collections, b_temp_path, apis,
     return dependency_map
 
 
+def _collections_from_scm(collection, requirement, b_temp_path, force, parent=None):
+    """
+    Returns a list of collections found in the repo. If there is a galaxy.yml in the collection then just return
+    the specific collection. Otherwise, check each top-level directory for a galaxy.yml.
```

Comment:
If it's meant to be compliant w/ PEP257, it should be:
```suggestion
    """Return a list of collections found in the repo.

    If there is a galaxy.yml in the collection then just return
    the specific collection. Otherwise, check each top-level
    directory for a galaxy.yml.
```